doc/run_experiment.rst

```diff
@@ -1676,6 +1682,28 @@ Here's an example of this plot.
 You can also generate the plots from the learning curve summary
 file using the :ref:`plot_learning_curves <plot_learning_curves>` utility script.
 
+.. _output_wandb:
+
+Weights and Biases
+^^^^^^^^^^^^^^^^^^
```

Comment:
I don't care that much about this aspect, but it would be good to use the name as it appears on their website and to use that same name everywhere. Abbreviations would be nice if they were also standardized (see comment elsewhere).
```suggestion
Weights & Biases
^^^^^^^^^^^^^^^^
```